src/components/Trophy.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components/native'
+
+import { TrophyIcon } from '../../assets/images'
+
+const TrophyContainer = styled.View`
+  margin-top: 11px;
+  display: flex;
+  flex-direction: row;
+`
+
+const TrophyIconStyled = styled(TrophyIcon)`
+  margin-right: 5px;
+`
+
+interface PropsType {
+  level: number
+}
+
+const Trophy = ({ level }: PropsType): ReactElement => {
+  const trophies = []
+  for (let i = 0; i < level; i += 1) {
+    trophies.push(<TrophyIconStyled key={`trophy-${i}`} testID={`trophy-${i}`} width={15} height={15} />)
```

Comment:
Just curious. Why didn't you move width and height into the component style specifications above?

Reply:
Because it did not work, not sure why.